packages/interactivity/src/vdom.ts

```diff
@@ -127,18 +131,22 @@ export function toVdom( root ) {
 							SCRIPT_DEBUG === true
 						) {
 							// eslint-disable-next-line no-console
-							console.warn( `Invalid directive: ${ name }.` );
+							console.warn(
+								`Invalid directive: ${ directiveName }.`
+							);
 						}
 						return obj;
 					}
-					const prefix = directiveMatch[ 1 ] || '';
-					const suffix = directiveMatch[ 2 ] || 'default';
-
-					obj[ prefix ] = obj[ prefix ] || [];
-					obj[ prefix ].push( {
-						namespace: ns ?? currentNamespace(),
-						value,
-						suffix,
+					// Splitting the directive name into directive type and input.
+					const directiveType = directiveMatch[ 1 ] || '';
```

Comment:
```suggestion
					const directive = directiveMatch[ 1 ] || '';
```
I think directive is a better name.